avocado/core/sysinfo.py

```diff
@@ -161,8 +162,24 @@ def run(self, logdir):
         stdin = open(os.devnull, "r")
         stdout = open(logf_path, "w")
         try:
-            subprocess.call(self.cmd, stdin=stdin, stdout=stdout,
-                            stderr=subprocess.STDOUT, shell=True, env=env)
+            proc = subprocess.Popen(self.cmd, stdin=stdin, stdout=stdout,
+                                    stderr=subprocess.STDOUT, shell=True,
+                                    env=env)
+            deadline = time.time() + 10
+            while time.time() < deadline:
+                if proc.poll() is not None:
+                    break
+                time.sleep(0.01)
```

Comment:
The problem is that `avocaod.utils.process` was misbehaving in `sysinfo`, anyway I can give it a try...